dynamic_rest/fields/fields.py

```diff
@@ -275,9 +275,17 @@ def to_representation(self, instance):
         serializer = self.serializer
         model = serializer.get_model()
         source = self.source
-        if not self.kwargs['many'] and serializer.id_only():
+        try:
+            lookup_field = serializer.Meta.lookup_field
```

Comment:
The Meta attribute is not always available on the Serializer so the try/except is used for two levels instead of one